test/clj/rems/test_browser.clj

```diff
@@ -407,7 +410,53 @@
       (is (btu/visible? {:css "div.event-description b" :fn/text "Developer approved the application."})))
     (testing "attachments visible in eventlog"
       (is (= ["test.txt" "test-fi.txt"]
-             (get-attachments {:css "div.event a.attachment-link"}))))))
+             (get-attachments {:css "div.event a.attachment-link"}))))
+    (testing "event via api"
+      (is (= {:application/id (btu/context-get :application-id)
+              :event/type "application.event/approved"
+              :application/comment "this is a comment"
+              :event/actor "developer"}
```

Comment:
note: no `:entitlement/end` here

Reply:
could perhaps put it in a commented line? it's rather implicit now. or in the `testing` clause as `event via api does not have entitlement end date`